articles/service-bus-messaging/service-bus-java-how-to-use-queues.md

```diff
@@ -216,11 +216,9 @@ See the following documentation and samples:
 
 - [Azure Service Bus client library for Java - Readme](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/servicebus/azure-messaging-servicebus/README.md)
 - [Samples on GitHub](https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/servicebus/azure-messaging-servicebus/src/samples)
-- [Java API reference](/java/api/overview/azure/servicebus?preserve-view=true&view=azure-java-preview)
-
-See [more samples on GitHub](https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/servicebus/azure-messaging-servicebus). 
```

Comment:
We have already specified the samples at line 218, so we do not need this link again.